source/modes/ConsoleCommands.cpp

```diff
@@ -530,6 +536,11 @@ void addConsoleCommands(ConsoleInterface& cl)
                   cSrvAddCreature,
                   {AbstractModeManager::ModeType::GAME, AbstractModeManager::ModeType::EDITOR});
 
+    // cl.addCommand("startTileCulling",
```

Comment:
I would lovely have both startTileCulling and stopTileCulling . However I cannot cope with the code ; could you correct the code below : 

```
    cl.addCommand("startTileCulling",
                  "start culling of the tiles",
                  cStartTileCulling,
                  {AbstractModeManager::ModeType::GAME, AbstractModeManager::ModeType::EDITOR});
```